### PR TITLE
ADD drag handles + MINOR refactoring

### PR DESCRIPTION
**Pull request by haraldox**
@bokuweb GREAT WORK SO FAR!!!  :+1: 
here are some minor improvements, hope you're willing to pull them after reviewing ;)

ADD prop `draggableHandleClass` and change `onMouseDown` handler to drag (sort) panes only when dragging target has specific className :)
[MINOR] REFACTOR `next` -> `nextProps`
[MINOR] ADD .gitignore

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,7 +79,6 @@
   ],
   "dependencies": {
     "lodash.isequal": "^4.1.1",
-    "react-motion": "^0.3.1",
-    "react-resizable-box": "^0.3.3"
+    "react-motion": "^0.3.1"
```
Comment (bokuweb) [inline package.json:82]: Sorry. please add `react-resizable-box` to dependencies.

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import { Motion, spring } from 'react-motion';
-import Resizable from 'react-resizable-box';
+import Resizable from '/lib/react-resizable-box/src/index.js';
```
Comment (bokuweb) [inline src/index.js:3]: Please fix to `import Resizable from 'react-resizable-box';`

**Comment by bokuweb** [comment]
@haraldox tahnks PR!!! 
I'm sorry, but please load `react-resizable-box` from `npm`.

**Comment by haraldox** [comment]
@bokuweb I will, just let me set up another branch, so I don't break my own dev environment and build process, thanks :)

**Comment by haraldox** [comment]
@bokuweb closing in favor of #37

**Pull request closed without merging**